client/src/main/java/org/spine3/validate/Validate.java

```diff
@@ -107,10 +111,57 @@ public static boolean isNotDefault(Message object) {
      * @param object the {@code Message} instance to check
      * @throws IllegalStateException if the object is in its default state
      */
+    public static <M extends Message> M checkNotDefaultState(M object) {
+        checkNotNull(object);
+        checkNotDefaultState(object,
+                             "The message is in the default state: %s",
+                             TypeName.of(object));
+        return object;
+    }
+
+    /**
+     * Ensures that the passed object is not default and is not {@code null}.
+     *
+     * @param object    the {@code Message} instance to check
+     * @param errorMsg  the message for the exception to be thrown;
+     *                  will be converted to a string using {@link String#valueOf(Object)}
+     * @throws IllegalArgumentException if the object is in its default state
+     */
+    public static <M extends Message> M checkNotDefault(M object, @Nullable Object errorMsg) {
```

Comment:
Please avoid having `@Nullable` method arguments in the public API. This method should have an overloading with a single argument.